src/vs/workbench/api/common/extHostDecorations.ts

```diff
@@ -106,7 +106,7 @@ export class ExtHostDecorations implements ExtHostDecorationsShape {
 					}
 					result[id] = <DecorationData>[data.propagate, data.tooltip, data.badge, data.color];
 				} catch (e) {
-					this._logService.warn(`INVALID decoration from extension '${extensionId}': ${e}`);
```

Comment:
You're just deleting the warning, not fixing it as you suggest in the comment?

I'd like to know what extension is giving me these warnings, not just suppress it.

Reply:
see the green line below